docs/how-to-guides/streams/kafka/connect-to-stream-from-lab.md

```diff
@@ -0,0 +1,89 @@
+---
+id: connect-to-stream-from-lab
+title: How to connect to a data stream from Memgraph Lab
+sidebar_label: Connect to a data stream from Memgraph Lab
+---
+
+If you prefere to use GUI, you can now connect to data streams by using a wizard
+in the Memgraph Lab.
+
+If you need a stream to play around with, we've provided some [Awesome
+Streams](https://awesomedata.stream/)! 
+
+## 1. Add a stream
+
+To add a stream in Memgraph Lab: 
+
+1. Switch to **Streams** and **Add New Stream**.
+2. Choose Kafka stream type, enter a stream name, server address, and topics you want to subscribe to.
+3. Go to the **Next Step**.
+
+If you are trying to connect to MovieLens data stream, the stream configuration
+should looks like this:
+

```

Comment:
Configuration is missing. We could add a screenshot. What say you?

Reply:
we can comment it for now and when we get the picture uncomment

Reply:
now i can change this because the lab won't be changed in this regard